client/lib/domains/dns/index.js

```diff
@@ -5,22 +5,23 @@ const includes = require( 'lodash/collection/includes' ),
 	mapValues = require( 'lodash/object/mapValues' ),
 	endsWith = require( 'lodash/string/endsWith' );
 
-function validateAllFields( fieldValues ) {
+function validateAllFields( fieldValues, domainName ) {
 	return mapValues( fieldValues, ( value, fieldName ) => {
 		const isValid = validateField( {
 			name: fieldName,
 			value: value,
 			type: fieldValues.type,
+			domainName: domainName
```

Comment:
I thought about it at first too, but coming from languages like Java, it feels weird to me to rearrange the parameters to a function like that (because our guidelines require shorthands to be at the top of the list). But I guess not using a shorthand could also be confusing, because someone could think that I'm mapping `domainName` to something else than `domainName`.
So shorthand it is! :laughing: